azure-container-apps/ReadMe.md

```diff
@@ -341,34 +353,35 @@ Logs for a container app can be viewed from the **Log Stream**, there is a separ
     az containerapp revision restart -n $ConAppName -g $ResourceGroup --revision revisionName
     ```
 
-4. Test logging into your SCIM bridge URL with the new `bearer token`. 
+4. Open your SCIM bridge URL in a browser and enter your bearer token to test the bridge.
+
 5. Update your identity provider configuration with the new bearer token.
-</details>
+    </details>
 
 <details>
 <summary>Replace your `scimsession` secret using the Azure Portal</summary>
 
-1. Get the Base64 encoded contents of your newly downloaded `scimsession` file, _(using the bash or PowerShell syntax for the commands)_: 
+1. Run the following command for your shell to get the Base64 encoded contents of your new `scimsession` file.
 
-    - Using Bash (Cloud Shell, macOS, or Linux):
+    - **Bash**:
 
         ```bash
         cat ./scimsession | base64
         ```
 
-    - Using PowerShell (Cloud Shell, Windows, macOS, or Linux):
+    - **PowerShell**:
 
         ```pwsh
         [Convert]::ToBase64String([IO.File]::ReadAllBytes((Join-Path $PWD.Path 'scimsession')))
         ```
 
-    Copy the output value from the terminal to your clipboard. It will be needed to create the secret for the deployment.
-
-2. Within Container App from the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps), select **Secrets** from the sidebar on the left hand side. 
-3. Edit the `scimsession` secret and paste the new value of your secret.
-4. Select the checkbox and click **Save**.
-5. Select the **Revisions** from the sidebar on the left hand side. 
-6. Click your current active revision, select Restart along the top. 
-7. Test logging into your SCIM bridge URL with the new `bearer token`.
-8. Update your identity provider configuration with the new bearer token.
-</details>
+2. Copy the output value to your clipboard to use it in the deployment.
+3. Open the Azure Portal and go to the [Container Apps](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps) page.
+4. Choose **Secrets** from the Settings section in the sidebar.
+5. Edit the `scimsession` secret and paste the new value of your secret.
+6. Select the checkbox and click **Save**.
+7. Choose the **Revisions** from the Application section in the sidebar.
+8. Click your current active revision and choose **Restart** in the details pane.
+9. Open your SCIM bridge URL in a browser and enter your bearer token to test the bridge.
+10. Update your identity provider configuration with the new bearer token.
```

Comment:
I left this as is, but am not really sure what it means. They aren't using their bearer token to connect their IdP...

Reply:
I am confused by this comment. When the credentials are regenerated in 1Password, both the scimsession secret on the deployment and their provisioning connection from their IdPs (except GW) require the enterprise/business application be updated with the new bearer token to continue functioning with the new scimsession.